### PR TITLE
Add basic contribution documentation

### PR DESCRIPTION
**Pull request by stevendanna**

### CODE REVIEW

**Comment by stevendanna** [comment]
cc @chef/lob

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,78 @@
...
+[Contributor License Agreement (CLA)](https://supermarket.getchef.com/become-a-contributor).
+
+2. Create a GitHub Pull Request.
+
+3. If you are a Chef Software employee, submit an ad-hoc
+chef-server-12 build for your branch and post a link to in the pull
+request. For non Chef Software employees, your job will be submitted
+for you by one of the reviewers.
+
+4. Respond [Code Review](#cr) to code review comments to improve the
```
Comment (danielsdeleo) [inline CONTRIBUTING.md]: This reads as "Respond Code Review to code review comments," probably not what you intended ;)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,78 @@
...
+- What you actually see given the steps you've described.
+
+## <a name="cr"></a> Code Review Process
+
+The Chef Code Review process happens on Github pull requests. See
+[this article](https://help.github.com/articles/using-pull-requests)
+if you're not familiar with Github Pull Requests.
+
+Once you a pull request, the **Chef Engineering Team** or **Chef Core
+Committers** will review your code and respond to you with any
```
Comment (danielsdeleo) [inline CONTRIBUTING.md]: Is there a maintainer team for Chef Server that isn't "Chef Core Committers"? Does chef-core even have commit access here?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1 +1,78 @@
...
+   the change was made.
+
+3. Leave an initial pull request comment with any additional
+   information that you think might help reviewers but isn't
+   appropriate for commit messages.
+
+The process at this point is as follows:
+
+1. 2 thumbs-ups are required from the **Chef Engineering Team** or
+**Chef Server Committers** for all merges.
```
Comment (danielsdeleo) [inline CONTRIBUTING.md]: Same as the above is this the correct team?

**Comment by danielsdeleo** [comment]
:+1: pending a little cleanup per the inline comments.

File: CONTRIBUTING.md (hunk 2 above)
Comment (stevendanna) [inline CONTRIBUTING.md]: Ahh, meant to change this to "Chef Server Committers"

**Comment by marcparadise** [comment]
:+1: post cleanup

**Pull request merged**